gui/mozregui/log_report.py

```diff
@@ -27,15 +46,44 @@ def on_log_received(self, data):
         cursor_to_add = QTextCursor(message_document)
         cursor_to_add.movePosition(cursor_to_add.End)
         cursor_to_add.insertText(log_message + '\n')
+
         if data['level'] in COLORS:
             fmt = QTextCharFormat()
             fmt.setForeground(COLORS[data['level']])
             cursor_to_add.movePosition(cursor_to_add.PreviousBlock)
+            log_lvl_data = LogLevelData(log_levels[data['level'].upper()])
```

Comment:
I wonder if you should store directly the numeric value in there, it would be more efficient (you won't have to do the conversion each time a level change) ?